langkit/compiled_types.py

```diff
@@ -2512,6 +2512,16 @@ def __init__(self, name, location, doc, base, fields,
         entity_type = self.entity
         del entity_type
 
+    @property
+    def inherited_env_spec(self):
```

Comment:
This name is misleading, as the result is not always the inherited spec. What about `effective_env_spec` instead?

Reply:
Right, I agree about that. I don't much like `effective_env_spec` either. I think the proper fix would be to call the property `env_spec`, and rename the field something like `explicit_env_spec`. But that's a bigger refactoring.

Reply:
Fair enough about the bigger refactoring. Can you elaborate why you don’t like `effective_env_spec`?